service/dynamodb/expression/readme.md

```diff
@@ -0,0 +1,94 @@
+# expression
```

Comment:
This file's content should be apart of the doc.go file . This file as it is will never be visible to users when using godoc, or similar tools.

A new file added to the https://github.com/awsdocs/aws-go-developer-guide repo would be a good place to put extra higher level documentation, concepts, and detailed example based information. But `About DynamoDB Expressions` and `Using the Package` sections should be in the doc.go file regardless.